tests/manage/mcg/test_verify_noobaa_status.py

```diff
@@ -1,23 +1,41 @@
 import logging
+import re
 
 from ocs_ci.framework.pytest_customization.marks import tier1
-from ocs_ci.framework.testlib import polarion_id, bugzilla
+from ocs_ci.framework.testlib import polarion_id, bugzilla, version
 from ocs_ci.framework.pytest_customization.marks import skipif_openshift_dedicated
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import get_pod_logs
 
 log = logging.getLogger(__name__)
 
 
 @tier1
 @polarion_id("OCS-2084")
 @bugzilla("1799077")
+@bugzilla("2004130")
 @skipif_openshift_dedicated
 def test_verify_noobaa_status_cli(mcg_obj_session):
     """
     Verify noobaa status output is clean without any errors using the noobaa cli
     """
+    ocs_version_semantic = version.get_semantic_ocs_version_from_config()
+
     # Get noobaa status
     status = mcg_obj_session.exec_mcg_cmd("status").stderr
     for line in status.split("\n"):
         if "Not Found" in line:
             assert "Optional" in line, f"Error in noobaa status output- {line}"
+        if ocs_version_semantic >= version.VERSION_4_7 and "noobaa-db" in line:
```

Comment:
BZ 2004130 looks to be fixed in 4.8, should it be > 4.7 ?

Reply:
True, ack